src/test/java/net/minecraftforge/test/ItemFishedTest.java

```diff
@@ -0,0 +1,39 @@
+package net.minecraftforge.test;
+
+import net.minecraft.init.Items;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.entity.player.ItemFishedEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.event.FMLInitializationEvent;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+
+import java.util.List;
+
+@Mod(modid = "itemfishtest", name = "ItemFishTest", version = "1.0.0")
+public class ItemFishedTest
+{
+
+    private static final boolean ENABLE = true;
+
+    @Mod.EventHandler
+    public void onInit(FMLInitializationEvent event)
+    {
+        if (ENABLE)
+        {
+            System.out.println("Enabling Fishing Test mod");
```

Comment:
Please use the logger you get from `FMLPreInitializationEvent#getModLog` instead of `System.out`